daphne/src/messages.rs

```diff
@@ -809,6 +817,218 @@ impl Decode for HpkeCiphertext {
     }
 }
 
+#[derive(Clone, Deserialize, Serialize, Debug, PartialEq, Eq)]
+pub enum VdafType {
+    Prio3AES128Count,
```

Comment:
rust naming convention (clippy might complain about this, I'm not sure)
```suggestion
    Prio3Aes128Count,
```

Reply:
fixed in next push